### DISCUSSION TITLE
Attachment links to intermal fileserver

### DISCUSSION THREAD

**Discussion by xeno27** [inventree/InvenTree#5178] [Q&A] [answered]
Hi,

Iam a bit struggeling by adding an Attachment link to internal server file share like:

file:///blabla.pdf 

this end up in a failure : 
![Screenshot from 2023-07-05 11-00-43](https://github.com/inventree/InvenTree/assets/20142175/edcbe88c-1818-4c44-9ac8-9d99b2cce164)

is there any workaround ? because this is needed :/
if you help me, i will change this :)

**Comment by matmair** [comment, marked as answer]
We do not support the special file link-format - it is not an URL (see the 3 slashes). In any somewhat modern browser those won't work either way due to security reasons.

**Reply by xeno27** [reply]
Make sense from the security side. 
So there is maybe just the option to add a parameter with the file path. 
And copy this into the explorer ..

**Reply by matmair** [reply]
That would be a solution. Connecting on-prem and online is always a bit difficult. If you are in a windows AD network and have Intune/an endpoint management solution email me - I might have a few tips.
